api/provider/aws/common.go

```diff
@@ -243,3 +243,47 @@ func listClusterTaskARNs(ecsapi ecsiface.ECSAPI, clusterName, startedBy, status
 
 	return taskARNs, nil
 }
+
+func listContainerInstances(ecsapi ecsiface.ECSAPI, clusterName string) ([]string, error) {
```

Comment:
why is this is `common.go` - won't this only be used by the scaler?

Reply:
@sesh-kebab

Reply:
Changes to common can be removed. It doesn't look like the latest set of changes that @nehayward and @tlake worked on use these methods.